index.html

```diff
@@ -37,7 +38,180 @@ <h1 class="header"><span>Hey there.
+                        <li><img src="img/css.png" alt="css"></li>
+                        <li><img src="img/javascript.png" alt="javascript"></li>
+                        <li><img src="img/html.png" alt="html"></li>
+                    </ul>
+                    <button class="btn" type="button"><span class="text">See project</span></button>
+                </div>
+            </div>
+            <section id="about-id">
+                <h1 class="header-2">About me</h1>
+
+
+                <p class="paragraph"> I can help you build a product , feature or website Look through some of my work
+                    and
+                    experience! If you like
+                    what you see and have a project you need coded,
+                    don’t hesitate to contact me.
+                </p>
+
+                <button class="btn-1" type="button"><span class="text">Get my resume</span></button>
+            </section>
+            <div class="grid-item-1">
+                <div>
+                    <div id="one"></div>
+                    <h4 class="header-3">Languages</h4>
+                    <ul class="image-container-1">
+                        <li><img src="img/js.png" alt="javascript"></li>
+                        <li><img src="img/rub.png" alt="ruby"></li>
+                        <li><img src="img/ht.png" alt="html"></li>
+                        <li><img src="img/cs.png" alt="css"></li>
+                    </ul>
+                </div>
+            </div>
+            <div class="grid-item-1">
+                <div>
+                    <div id="two"></div>
+                    <h4 class="header-3">Frameworks</h4>
+                    <ul class="image-container-1">
+                        <li><img src="img/react.png" alt="React"></li>
+                        <li><img src="img/rails.png" alt="ruby"></li>
+                        <li><img src="img/rspec.png" alt="Rspec"></li>
+                        <li><img src="img/capy.png" alt="Capybara"></li>
+                        <li><img src="img/selenium.png" alt="Selenium"></li>
+                    </ul>
+                </div>
+            </div>
+            <div class="grid-item-1">
+                <div>
+                    <div id="three"></div>
+                    <h4 class="header-3">Skills</h4>
+                    <ul class="image-container-1">
+                        <li><img src="img/database.png" alt="database"></li>
+                        <li><img src="img/git.png" alt="git"></li>
+                        <li><img src="img/cli.png" alt="cli"></li>
+                        <li><img src="img/web.png" alt="web"></li>
+                        <li><img src="img/api.png" alt="api"></li>
+                    </ul>
+                </div>
+            </div>
+            <div>
+                <h1 class="header-4">I'm always interested in hearing about new projects, so if you'd like to chat
+                    please get in touch.</h1>
+            </div>
```

Comment:
Please kingly implement the following design changes for the `about` section of your project :

- kindly add the proper margin for this section in your project so as to enhance your design interface
- please use the right font color of  `#42526E` in your paragraph `I can help you build` 😄 
- use the right font color of `#344563` for ` Languages` as it is in the design 👍 

Figma design:
![image](https://github.com/NgachaIan/Personal_Portfolio/assets/20537712/ca2b2659-887c-48f1-8661-64a14e337e5d)

Your design :
![image](https://github.com/NgachaIan/Personal_Portfolio/assets/20537712/5c1e8fa6-048f-43b1-9631-8d4b49779214)